dvc/output/gdrive.py

```diff
@@ -0,0 +1,7 @@
+from dvc.output.base import BaseOutput
+
+from ..tree.gdrive import GDriveTree
+
+
+class GDriveOutput(BaseOutput):
+    TREE_CLS = GDriveTree
```

Comment:
GDriveOutput is never used. We only need the Dependency for import-url.

Reply:
Done!